backend/Testing/Services/UtilsTests.cs

```diff
@@ -0,0 +1,17 @@
+using Shouldly;
+using Testing.SyncReverseProxy;
+
+namespace Testing.Services;
+
+public class UtilsTests
+{
+    [Theory]
+    [InlineData(nameof(SendReceiveServiceTests.ModifyProjectData), "modify-project-data")]
+    [InlineData(nameof(SendReceiveServiceTests.SendNewProject), "send-new-project")]
```

Comment:
Because I want to be able to generate project codes so that:
1) They're somewhat standardized
2) I don't have to think about them ever again (that's the best case scenario 🤣)

It's a bit goofy having a test to test a test util. I know. But it's so tiny.